Emphasise the use of github.action_path (#22464)
github/docs: content/actions/creating-actions/creating-a-composite-action.md

          shell: bash
    ```
    {% endraw %}

>   This file defines the `who-to-greet` input, maps the random generated number to the `random-number` output variable, add `github.action_path` to $GITHUB_PATH (in order to locate `goodbye.sh` file during execution) and runs the `goodbye.sh` script. It also tells the runner how to execute the composite action.

Reviewer: ```suggestion
  This file defines the `who-to-greet` input, maps the random generated number to the `random-number` output variable, adds the action's path to the runner system path (to locate the `goodbye.sh` script during execution), and runs the `goodbye.sh` script.
```
